lib/pages/player/player_item_panel.dart

```diff
@@ -543,11 +543,11 @@ class _PlayerItemPanelState extends State<PlayerItemPanel> {
                     ),
                   ),
                 ),
-          // 自定义顶部组件
-          Positioned(
+            // 自定义顶部组件
+            Positioned(
             top: 0,
-            left: 0,
-            right: 0,
+            left: MediaQuery.of(context).padding.left,
+            right: MediaQuery.of(context).padding.right,
```

Comment:
```suggestion
            left: 0,
            right: 0,
```

同上，在 `EmbeddedNativeControlArea` 内部用 `SafeArea`

```dart
child: EmbeddedNativeControlArea(
  requireOffset: !videoPageController.isFullscreen,
  child: SafeArea(
    top: false,
    bottom: false,
    left: videoPageController.isFullscreen,
    right: videoPageController.isFullscreen,
    child: MouseRegion(
```